### PR TITLE
[audio] audio tiptap settings

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

File: packages/plugin-audio/web/src/modals/AudioSettings.tsx [hunk 1]
```diff
@@ -262,7 +286,7 @@ const AudioSettings = ({
           fixed
           theme={theme}
           cancel={handleClose}
-          save={helpers.closeModal}
+          save={(helpers.closeModal || onSave) as () => void}
```
Comment (tamrat-bay) [inline packages/plugin-audio/web/src/modals/AudioSettings.tsx:289]: onDoneClick & onCancelClick

File: packages/plugin-audio/web/src/darft-convertors.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import type { DraftAudioData } from './types';
```
Comment (tamrat-bay) [inline packages/plugin-audio/web/src/darft-convertors.ts:1]: Remove File

File: packages/plugin-audio/web/src/types.ts [hunk 3]
```diff
@@ -19,3 +23,18 @@ export interface AudioPluginEditorConfig extends EditorPluginConfig {
 export interface AudioPluginViewerConfig extends ViewerPluginConfig {
   [key: string]: any; // eslint-disable-line @typescript-eslint/no-explicit-any
 }
+
+export interface DraftAudioData {
```
Comment (tamrat-bay) [inline packages/plugin-audio/web/src/types.ts:27]: Remove Type

File: packages/plugin-audio/web/src/editor.ts [hunk 4]
```diff
@@ -8,16 +8,23 @@ import { tiptapExtensions } from './tiptap/tiptap';
 import type { TiptapEditorPlugin } from 'ricos-tiptap-types';
 import { getToolbarButtons } from './getToolbarButtons';
 import { getAddButtons } from './getAddButtons';
+import { AudioPluginService } from './toolbar/audioPluginService';
+
+const audioPluginService = new AudioPluginService();
```
Comment (danarish4) [inline packages/plugin-audio/web/src/editor.ts:13]: lets move it inside `getToolbarButtons`

**Review by danarish4 — APPROVED** [review]

**Pull request merged**